### PR TITLE
Adds two new UI styles: Slimecore and Operative

### PR DESCRIPTION
**Pull request by TrustyGun**
Slimecore is a pale green palette, while Operative is black and red.

**Pictures**:

_Slimecore_
slimecore

_Operative_
operative

:cl:
rscadd: Adds two new UI styles, Slimecore and Operative.
rscadd: Slimecore is a pale green.
rscadd: Operative is black and red.
/:cl:

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
0/10
Make the operative hud style actually operative instead of recoloured Midnight.

(Love slimecore though)

**Comment by WJohn** [comment]
Looks pretty good. Although operative looks pretty similar to plasmafire.

**Comment by Fox-McCloud** [comment]
Why not just do what Bay does and have a white-colored UI that can be set to any color?

**Comment by RemieRichards** [comment]
@Fox-McCloud Incoming is actually working on that.

**Comment by Iamgoofball** [comment]
we really need new UI designs

not just recolors

**Comment by Anonus** [comment]
Operative's red is a bit too orange, tbh.

**Comment by PKPenguin321** [comment]
I like Fox's idea more. This is cool, but not as cool as his idea.

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -192,7 +192,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	//Sanitize
 	ooccolor		= sanitize_ooccolor(sanitize_hexcolor(ooccolor, 6, 1, initial(ooccolor)))
 	lastchangelog	= sanitize_text(lastchangelog, initial(lastchangelog))
-	UI_style		= sanitize_inlist(UI_style, list("Midnight", "Plasmafire", "Retro"), initial(UI_style))
+	UI_style		= sanitize_inlist(UI_style, list("Midnight", "Plasmafire", "Retro","Slimecore"
```
Comment (Anonus) [inline code/modules/client/preferences_savefile.dm]: I think you're missing an Operative here.

**Comment by Incoming5643** [comment]
Yes I'm already working on customizable UI styles. It won't be ready before the freeze though. If people like these UI's they can still be added in the mean time, converting them to the system I'm working on later would be very easy.

**Comment by WJohn** [comment]
While it looks pretty nice, incoming, I gotta say that there's not enough contrast between individual shades there on your midnight style mimic. Easy fix, though.

**Comment by Incoming5643** [comment]
yeah like I said, not done

**Comment by KazeEspada** [comment]
I love that operative.

**Comment by TrustyGun** [comment]
Slimecore is now transparent. Operative looks all around better, and both of them got tooltips.

Pictures in the OP are changed to reflect this.

**Comment by Anonus** [comment]
Slimecore isn't the same transparency as operative, though, probably because it's not plasmafire-based; plasmafire(and retro but retro is shit) has much lower transparency than midnight does.

**Comment by KazeEspada** [comment]
Can you bring back the orange operative? I liked it.

**Comment by optimumtact** [comment]
:+1: welcome to the repo also

**Comment by Steelpoint** [comment]
Unique coloured UI's for individual antagonists when?

**Comment by KazeEspada** [comment]
@steelpoint thats a shit idea

**Comment by Thunder12345** [comment]
Rather than just recoloured midnight, consider changing the syndicate UI to have syndie themes icons. Things like a syndie hardsuit for the suit slot, hardsuit helmet for the helmet show, syndie mask for the mask slot, and a captain for the targeting doll.

**Comment by Steelpoint** [comment]
Kaze, it would add to the overall immersion in the role. I'm sure if anyone ever did such a mad thing that they would allow you to disable antagonist unique UI colours/scheme's so you can use your default one.

**Comment by RemieRichards** [comment]
@Thunder12345 not even mentioning that the captain doll was my idea? thief!

**Comment by TrustyGun** [comment]
conflicts ffs

**Comment by optimumtact** [comment]
rip

**Comment by optimumtact** [comment]
come in irc #coderbus on rizon and mergebeg when you have fixed conflicts, best of luck

**Comment by Cheridan** [comment]
Doesn't seem to have worked

**Comment by TrustyGun** [comment]
I hope this works

**Comment by optimumtact** [comment]
@Cheridan free this person

**Comment by TrustyGun** [comment]
save me from the dark

**Pull request merged**